python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -63,28 +70,39 @@ def get_nodes_to_launch(
         node_resources, node_type_counts = \
             self.calculate_node_resources(nodes, pending_nodes, usage_by_ip)
 
+        logger.info("Cluster resources: {}".format(node_resources))
+        logger.info("Node counts: {}".format(node_type_counts))
+
         node_resources, node_type_counts, min_workers_nodes_to_add = \
             _add_min_workers_nodes(
                 node_resources, node_type_counts, self.node_types)
 
-        nodes_to_add_based_on_demand = {}
-        if resource_demands is not None:
-            logger.info("Cluster resources: {}".format(node_resources))
-            logger.info("Node counts: {}".format(node_type_counts))
-            unfulfilled = get_bin_pack_residual(node_resources,
-                                                resource_demands)
-            logger.info("Resource demands: {}".format(resource_demands))
-            logger.info("Unfulfilled demands: {}".format(unfulfilled))
-            max_to_add = self.max_workers - sum(node_type_counts.values())
-            nodes_to_add_based_on_demand = get_nodes_for(
-                self.node_types, node_type_counts, max_to_add, unfulfilled)
+        logger.info(f"Placement group demands: {placement_group_load}")
+        placement_group_demand_vector, strict_spreads = \
+            placement_groups_to_resource_demands(placement_group_load)
+        resource_demands += placement_group_demand_vector
```

Comment:
```suggestion
        resource_demands.extend(placement_group_demand_vector)
```

Please avoid symbolic operations when possible.